### ISSUE TITLE
Extend story video length 

### ISSUE THREAD

**Issue by tzuyu18** [haoict/messenger-no-ads#36]
Extend video story length doesn't work after 308.0 update. 
After the compression it goes back to messenger home
Tried reinstalling both tweak and the messenger but still not working.
Hope it will be fixed thanks for this wonderful tweak

---

### Environment
- Platform+version: **iOS 14.4.1** 
- Tweak version: **3.10.0** 
- Do you have jailbreak detection bypass tweaks (FlyJB, HideJB, kernbypass, ...): **Yes
- If yes, are you sure that it does not block this tweak: **Yes**

### Bug Report

Note: Providing steps to reproduce, screenshot or screen recording are always helpful!

**Comment by haoict** [comment]
Probably Facebook fixed it on server side, I'll check
Have you tried downgrade Messenger?

**Comment by tzuyu18** [comment]
Yes I downgraded for the previous version and it works so the latest version has the problem only. I blocked the update for the meantime. Thanks for the effort.

**Comment by theblues08** [comment]
Same to me,  downgraded version doesn’t work :(

**Comment by haoict** [comment]
I think it's a bug of Messenger itself. I tried uninstall/disable the tweak then upload a long video and it seemed not working

**Comment by tzuyu18** [comment]
> Same to me,  downgraded version doesn’t work :(

Use appstore++ to downgrade to version 307.2 or lower for the meantime.
But after adding the story it won't automatically close, so you should press back and discard. Now checked if the video is uploading on your story.
